documentation/changes.md

```diff
@@ -0,0 +1,68 @@
+# Breaking Change Policy
+
+Customers and MVPs have told us many times in emphatic terms that breaking changes are a continuing problem in Azure PowerShell releases.  These changes cause an undue burden on users maintaining scripts to manage their Azure assets, and this causes reluctance in picking up the later versions of Azure PowerShell. It is clear that continuing, unmanaged breaking changes are unnacceptable to customers. 
+
+**To address customer concerns, Azure PowerShell is committing to limit breaking changes to one release per year and to treat any unplanned breaking change that ships as 
+a showstopper bug requiring a hotfix**.
+
+At the same time, we are introducing a new mechanism for [Pre-release AzureRM modules](#preview-modules).  This is detailed in the section [here](#preview-modules).
+
+This proposed policy is meant to apply to all stable Azure modules, starting with the 2.0 release. Details of the policy are as follows.
+- Each module in Azure PowerShell is versioned using [semantic versioning](http://semver.org).
+- Non-breaking changes and associated version updates are handled independently by module owners.  [Breaking changes](#breaking-change-definition) are scheduled for once a year and coordinated with the Azure SDK Team.
+- Each breaking change release will be preceded by 3-months notice of the break to customers.  The SDK Team will coordinate the notice of breaking changes.
+- It is the responsibility of the service teams to introduce no breaking changes in their cmdlet external interfaces.  A definition of breaking changes is provided [here](#breaking-change-definition).  It is the responsibility of the SDK Team to introduce no breaking changes in SDK cmdlets, and in the public types in common code used as parameters or returned by cmdlets.
+  - The SDK Team will provide tools in the October release to automatically detect and prevent breaking changes
+  - Until breaking change detection tools are available, the feature teams and SDK team will use code reviews to prevent breaking changes
```

Comment:
@brendanburns  We try to.  It is difficult to catch all breaking changes this way, however, which is why tooling is necessary.